ledger-tool/src/args.rs

```diff
@@ -112,6 +112,23 @@ pub fn accounts_db_args<'a, 'b>() -> Box<[Arg<'a, 'b>]> {
     .into_boxed_slice()
 }
 
+// For our current version of CLAP, the value passed to Arg::default_value()
+// must be a &str. But, we can't convert an integer to a &str at compile time.
+// So, declare this constant and enforce equality with the following unit test
+// test_max_genesis_archive_unpacked_size_constant
+const MAX_GENESIS_ARCHIVE_UNPACKED_SIZE_STR: &str = "10485760";
+
+/// Returns the arguments that configure loading genesis
+pub fn load_genesis_args<'a, 'b>() -> Box<[Arg<'a, 'b>]> {
+    vec![Arg::with_name("max_genesis_archive_unpacked_size")
+        .long("max-genesis-archive-unpacked-size")
+        .value_name("NUMBER")
+        .takes_value(true)
+        .default_value(MAX_GENESIS_ARCHIVE_UNPACKED_SIZE_STR)
+        .help("maximum total uncompressed size of unpacked genesis archive")]
+    .into_boxed_slice()
+}
```

Comment:
Regarding the Vec, are you expecting more args to come for this list?

Reply:
> are you expecting more args to come for this list?

I am not, no. I did it for the sake of consistency with these:
https://github.com/anza-xyz/agave/blob/d2b3476db9951448b062319f3256a2a580bdf6a9/ledger-tool/src/args.rs#L29
https://github.com/anza-xyz/agave/blob/d2b3476db9951448b062319f3256a2a580bdf6a9/ledger-tool/src/args.rs#L116

That being said, I could see the argument for making it return single element, and then updating to return `Box<[_]>` if/when that day comes (again, nothing that I know of at the moment).

Reply:
> I could see the argument for making it return single element, and then updating to return Box<[_]> if/when that day comes (again, nothing that I know of at the moment).

That would be my inclination. I don't feel super strongly about it, though.

Reply:
Works for me - it is ledger-tool so we're pretty free to change the API (I might feel differently if this were a more public API). Done in https://github.com/anza-xyz/agave/pull/1774/commits/2f9e1f78d7f5897668ead4144182635d3030a800